notifications_utils/recipients.py

```diff
@@ -321,9 +321,18 @@ def _get_error_for_field(self, key, value):  # noqa: C901
                     email_address.validate_email_address(value)
                 if self.template_type == "sms":
                     if self.allow_sms_to_uk_landline:
-                        PhoneNumber(value, allow_international=self.allow_international_sms)
+                        number = PhoneNumber(
+                            value,
+                        )
+                        number.validate_phone_number(
+                            allow_international_number=self.allow_international_sms,
+                            allow_uk_landline=self.allow_sms_to_uk_landline,
+                        )
                     else:
-                        phone_number.validate_phone_number(value, international=self.allow_international_sms)
+                        phone_number.validate_phone_number(
+                            value,
+                            international=self.allow_international_sms,
+                        )
```

Comment:
Why are we still using the old way of doing things here?

Reply:
To be removed in the next ticket.